website/docs/docs/cloud/secure/ip-restrictions.md

```diff
@@ -1,10 +1,12 @@
 ---
 title: "Configuring IP restrictions"
 id: ip-restrictions
-description: "Configuring IP retrictions to outside traffic from accessing your dbt Cloud environment"
+description: "Configuring IP restrictions to outside traffic from accessing your dbt Cloud environment"
 sidebar_label: "IP restrictions"
 ---
 
+<Snippet src="available-tiers-iprestrictions" />
```

Comment:
Switch to partials?
>`import SetUpPages from '/snippets/_available-tiers-iprestrictions.md';`

Reply:
addressed in https://github.com/dbt-labs/docs.getdbt.com/pull/3616